docker/build_scripts/build.sh

```diff
@@ -10,14 +10,14 @@ MY_DIR=$(dirname "${BASH_SOURCE[0]}")
 
 # Dependencies for compiling Python that we want to remove from
 # the final image after compiling Python
-PYTHON_COMPILE_DEPS="zlib-devel bzip2-devel expat-devel ncurses-devel readline-devel tk-devel gdbm-devel db4-devel libpcap-devel xz-devel"
+PYTHON_COMPILE_DEPS="zlib-devel bzip2-devel expat-devel ncurses-devel readline-devel tk-devel gdbm-devel db4-devel libpcap-devel xz-devel openssl-devel keyutils-libs-devel krb5-devel libcom_err-devel libidn-devel curl-devel perl-devel"
```

Comment:
No "db4-devel" in CentOS 7. "libdb-devel" provides headers for Berkeley DB 5.3.21 version.

Reply:
I so this one one i686 now. I missed it on aarch64. it's present on x86_64.

Reply:
```
[root@c79c2af72ba7 /]# uname -m
x86_64
[root@c79c2af72ba7 /]# yum list db4-devel
Loaded plugins: fastestmirror, ovl
Loading mirror speeds from cached hostfile
 * base: mirror-pl.kielcetechnologypark.net
 * centos-sclo-rh: ftp.icm.edu.pl
 * centos-sclo-sclo: mirror-pl.kielcetechnologypark.net
 * extras: mirror-pl.kielcetechnologypark.net
 * updates: mirror-pl.kielcetechnologypark.net
Error: No matching Packages to list
```